src/Equinox.DynamoStore/Equinox.DynamoStore.fsproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\Equinox.Core\CachingDecorator.fs" Link="CachingDecorator.fs" />
```

Comment:
We already depend on Equinox Core in this Project, so this should be unnecessary

Reply:
Yeah, but it's `None` in the Core fsproj (and the helpers and types are/were intended to be internal). Main thing is that `Core.Caching` would be in the API/intellisense surface.
But yes, will likely remove this messing as the MemoryStore is the only store that does not use it anyway

Reply:
Caching.fs is now in Core